File: Poo/BoletinIntermedio/Sumando.java
import java.util.Scanner;

public class Sumando {
  private int NumeroUno;
  private int NumeroDos;
  
  public Sumando(){
      this.NumeroUno = 1 ;
      this.NumeroDos = 2 ;
  }

  public Sumando (int NumeroUno, int NumeroDos){
      this.NumeroUno = NumeroUno;
      this.NumeroDos = NumeroDos;
  }

  Scanner sc = new Scanner(System.in);

  public int getNumeroUno() {
    return NumeroUno;
  }
  public int getNumeroDos() {
    return NumeroDos;
  }

  public void setNumeroUno(int NumeroUno) {
    this.NumeroUno = NumeroUno;
  }
  public void setNumeroDos(int NumeroDos) {
    this.NumeroUno = NumeroDos;
  }

  public int Suma(int NumeroUno, int NumeroDos) {
    int suma = NumeroUno + NumeroDos;
    return suma;
  }

  public int leer() {
    return Integer.parseInt(this.sc.nextLine());
  }

  public static void main(String[] args) {
    Sumando sumando1 = new Sumando();
    System.out.println("Introduceme un numero");
    sumando1.setNumeroUno(sumando1.leer());
    System.out.println("Introduceme un numero");
    sumando1.setNumeroDos(sumando1.leer());
  }
}

File: Poo/Sumando.py
class Sumando :
    NumeroUno = 0
    NumeroDos = 0
    def __init__(self) :
        self.NumeroUno = 1
        self.NumeroDos = 2
    def __init__(self, NumeroUno,  NumeroDos) :
        self.NumeroUno = NumeroUno
        self.NumeroDos = NumeroDos
  
    def  getNumeroUno(self) :
        return self.NumeroUno
    def  getNumeroDos(self) :
        return self.NumeroDos
    def setNumeroUno(self, NumeroUno) :
        self.NumeroUno = NumeroUno
    def setNumeroDos(self, NumeroDos) :
        self.NumeroUno = NumeroDos
    def  Suma(self, NumeroUno,  NumeroDos) :
        suma = NumeroUno + NumeroDos
        return suma
    def  leer(self) :
        return int(input())
    @staticmethod
    def main( args) :
        sumando1 = Sumando()
        print("Introduceme un numero")
        sumando1.setNumeroUno(sumando1.leer())
        print("Introduceme un numero")
        sumando1.setNumeroDos(sumando1.leer())